package/TSAccountProtectionProvider.tsx

```diff
@@ -101,12 +105,14 @@ const getLoadSDKPromise = (id: string, sdkVersion: string, parentElement?: HTMLH
   });
 };
 
-const buildProviderState = (clientId: string, options?: { [key: string]: any }): ProviderState => {
+const buildProviderState = (clientId: string, options?: DRSConfigOptions): ProviderState => {
+  const sdkVersion = options?.sdkVersion ?? 'latest';
   return {
-    initialized: false,
+    initialized: new Promise((res) => undefined), // making default promise in pending state
     clientId,
-    serverUrl: options?.serverPath ?? 'https://collect.riskid.security/',
-    sdkVersion: options?.sdkVersion ?? 'latest',
+    serverPath: options?.serverPath ?? 'https://collect.riskid.security/',
```

Comment:
@LidanTransmit @ShaulTransmit is someone using this `serverPath` option?
i would like to change this to `serverUrl` like the angular wrapper

Reply:
Ethos are using server url, but you can update it and will let them know  you will need to change the major version as this breaking backward compatability